main.py

```diff
@@ -213,6 +256,12 @@ def _update_switch_flows(self, switch):
         del self._multipart_replies_flows[switch.id]
         del self._multipart_replies_xids[switch.id]['flows']
 
+    def _update_switch_tables(self, switch):
+        """Update controllers' switch tables list and clean resources."""
+        switch.tables = self._multipart_replies_tables[switch.id]
```

Comment:
It's not desirable to store `TableStats` on switch.tables, `kytos_stats` will be responsible to subscribing and storing it in memory (and in the future maybe in a DB), can you remove this part? Also, we probably don't even need this method and you could move the `del` statements to `_handle_multipart_table_stats` itself.